src/SFML.System/Time.cs

```diff
@@ -46,26 +46,49 @@ public struct Time : IEquatable<Time>
         ////////////////////////////////////////////////////////////
         public static Time FromMicroseconds(long microseconds) => sfMicroseconds(microseconds);
 
+        ////////////////////////////////////////////////////////////
+        /// <summary>
+        /// Construct a Time value from a TimeSpan
+        /// </summary>
+        /// <param name="timeSpan">A TimeSpan representing the amount of time to represent</param>
+        /// <returns>Time value constructed from an existing TimeSpan</returns>
+        ////////////////////////////////////////////////////////////
+        public static Time FromTimeSpan(TimeSpan timeSpan) => sfMicroseconds(timeSpan.Ticks / 10);
```

Comment:
I might be wrong, but I don't think, you can just convert between ticks and time like this, or can you?
[Ticks](https://docs.microsoft.com/en-us/dotnet/api/system.timespan.ticks?view=netstandard-2.0#System_TimeSpan_Ticks) doesn't seem to have a specific unit and I guess there's a reason why [`TimeSpan.TicksPerMillisecond`](https://docs.microsoft.com/en-us/dotnet/api/system.timespan.tickspermillisecond?view=netstandard-2.0) exists.

Reply:
.NET Standard has `TimeSpan.TicksPerMillisecond` as a const `10000`, making 1 microsecond = 10 ticks.

Reply:
Shouldn't you still use the const instead of writing your own hard coded values? And does the standard require this, or is that just one implementation of the standard?

Reply:
It is the Standard, the const is for clarity, but will actually require a division operation to do the milliseconds to microseconds conversion, as there is not a `TicksPerMicrosecond` field.  The implementation I've done here is virtually identical to SFML's native one.

Reply:
I'm pretty sure that the C# compiler can pre-compute the result of a constant divided by another constant, and not add extra runtime CPU cycles for it.

Reply:
I can switch this to use the `TicksPerMillisecond` constant, but it is currently identical to the native implementation, and also will not truncate accuracy as it is currently.

Reply:
> it is currently identical to the native implementation

Which native implementation are you referring to?

> will not truncate accuracy as it is currently

Where is it truncating accuracy?

Reply:
The native implementation in `sf::Time::` for converting microseconds to seconds/milliseconds.

The truncation of accuracy from using `TicksPerMillisecond` would result from microseconds between milliseconds being lost.

Reply:
> The native implementation in sf::Time:: for converting microseconds to seconds/milliseconds.

Milli/micro/seconds are SI units and their conversion factors are well known... On the other hand, system "ticks" may be anything, that's why they provide such a constant to perform conversions.

> The truncation of accuracy from using TicksPerMillisecond would result from microseconds between milliseconds being lost.

I don't how using a named constant rather than a numeric constant would change anything.

Reply:
Using milliseconds runs the number of ticks through a lower granularity.  Additionally, from [MSDN](https://docs.microsoft.com/en-us/dotnet/api/system.datetime?view=netframework-4.8)`Time values are measured in 100-nanosecond units called ticks.`

Reply:
Upon further review, I can change this to use the constant.  I will update my PR shortly.

Reply:
PR is updated and commits squashed.